src/main/java/mate/academy/spring/util/impl/HashUtilImpl.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.util.impl;
+
+import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
+import mate.academy.spring.util.HashUtil;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HashUtilImpl implements HashUtil {
```

Comment:
```suggestion
public class HashUtil {
```
we don't usually need interfaces for util logic